R/prep_scores_scorecard.R

```diff
@@ -117,7 +117,7 @@ prep_scores_exposure_scorecard <- function(audit_data,
                                            currency_exchange_value,
                                            total_portfolio_value_curr,
                                            log_dir = .GlobalEnv$log_dir) {
-  if (is.null(emissions) || is.null(currency_exchange_value) || is.null(total_portfolio_value_curr)) {
+  if (is.null(emissions_data) || is.null(currency_exchange_value) || is.null(total_portfolio_value_curr)) {
```

Comment:
this one should actually check `is.null(audit_data)`, not `is.null(emissions)` or `is.null(emissions_data)`

Reply:
thanks!

Reply:
done